api/src/main/java/org/apache/iceberg/io/FileIO.java

```diff
@@ -58,4 +58,11 @@ default void deleteFile(InputFile file) {
   default void deleteFile(OutputFile file) {
     deleteFile(file.location());
   }
+
+  /**
+   * make a path on file system
+   * @return
+   */
+  boolean mkdir(String path);
```

Comment:
I don't think that this should be part of `FileIO`. This operation is specific to Hadoop, so it is okay to use a Hadoop FileSystem for it. `FileIO` is intended to remain simple and focused on individual data files.

Reply:
thanks Ryan , now has fixed.